### PR TITLE
webui: fix disk selection not working when moving back to the disk selection screen

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

File: ui/webui/src/components/storage/InstallationDestination.jsx [hunk 1]
```diff
@@ -151,7 +151,7 @@ const DropdownBulkSelect = ({
           key="select-multiple-split-checkbox"
           id="select-multiple-split-checkbox"
           aria-label={_("Select multiple disks")}
-          isChecked={selectedCnt > 0 ? (selectedCnt === totalCnt ? true : null) : false}
+          isChecked={selectedCnt === totalCnt}
```
Comment (rvykydal) [inline ui/webui/src/components/storage/InstallationDestination.jsx:154]: So we want this UI change - to get rid of this bulk selection checkbox state (some disks (neither all nor none) selected in this PR)?
Screenshot from 2023-07-13 09-47-37

**Review by rvykydal — COMMENTED** [review]
Looks good to me if the UX change is intentional.

File: ui/webui/src/components/storage/InstallationDestination.jsx (hunk 1 above)
Comment (KKoukiou) [reply ui/webui/src/components/storage/InstallationDestination.jsx:154]: oops good catch, how the tests even passed here? I will remove this commit.

Comment (KKoukiou) [reply ui/webui/src/components/storage/InstallationDestination.jsx:154]: I think the checkbox should be indeed only checked if all are checked.

**Review by rvykydal — APPROVED** [review]
Thank you.

File: ui/webui/src/components/storage/InstallationDestination.jsx (hunk 1 above)
Comment (rvykydal) [reply ui/webui/src/components/storage/InstallationDestination.jsx]: > oops good catch, how the tests even passed here? I will remove this commit.

I am afraid I just didn't write a test for this case. I'll add it.

**Pull request merged**